pages/index.tsx

```diff
@@ -20,15 +22,26 @@ export interface ErrorMessageProps {
 }
 
 const HomePage = () => {
+  const [activeFilter, setActiveFilter] = useState("");
+  const [clients, setClients] = useState([]);
   const [errorMessage, setErrorMessage] = useState<ErrorMessageProps>();
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const [isOpen, setIsOpen] = useState<boolean>(false);
   const [timeEntries, setTimeEntries] = useContext(StoreContext).timeEntries;
 
+  const filteredTimeEntries = timeEntries.filter((timeEntry) =>
+    activeFilter === "" ? true : timeEntry.client === activeFilter,
```

Comment:
nice.

You could be more specific in the case of this Filter, as it only filters for the clients/
In the case of multiple filter options, the activeFilter is too non-descript